src/storage/txn/scheduler.rs

```diff
@@ -144,107 +146,164 @@ impl Scheduler {
 
     fn process_cmd_with_snapshot(&mut self, cid: u64, snapshot: &Snapshot) -> Result<()> {
```

Comment:
@zhangjinpeng1987 is working on moving it to a thread pool. I suggest we keep all old code untouched to prevent too many merge conflicts. So does the 2 comments below.